src/app/components/AssignmentForm.tsx

```diff
@@ -56,26 +71,55 @@ export interface AssignmentFormProps {
 
 }
 
-export default class AssignmentForm extends React.Component<AssignmentFormProps & InjectedFormProps<any,AssignmentFormProps>, any>{
+export default class AssignmentForm extends React.Component<AssignmentFormProps & InjectedFormProps<any, AssignmentFormProps>, any>{
 
     render() {
-        const { handleSubmit, showCourtSecurityFields, showDocumentSericesFields, showEscortServicesFields, showGateSecurityFields, showOtherAssignmentFields } = this.props;
+        const { handleSubmit, showCourtSecurityFields, showDocumentSericesFields, showEscortServicesFields, showGateSecurityFields, showOtherAssignmentFields, isDefaultTemplate } = this.props;
         return (
             <div>
-                { showCourtSecurityFields && <h1> Court Security </h1> }
-                { showDocumentSericesFields && <h1> Document Services </h1> } 
-                { showEscortServicesFields && <h1> Escort Services </h1> }
-                { showGateSecurityFields && <h1> Gate Security </h1> }
-                { showOtherAssignmentFields && <h1> Other Assignment </h1>}
+                {showCourtSecurityFields && <h1> Courts</h1>}
```

Comment:
Already chatted about this over the phone, having a separate `renderHeading` method would be good